Sources/PointFree/About.swift

```diff
@@ -8,15 +8,15 @@ import Prelude
 import Styleguide
 import Tuple
 
-let aboutResponse: Middleware<StatusLineOpen, ResponseEnded, Tuple3<Database.User?, Stripe.Subscription.Status?, Route?>, Data> =
+let aboutResponse: Middleware<StatusLineOpen, ResponseEnded, Tuple3<Database.User?, SubscriberState, Route?>, Data> =
```

Comment:
`SubscriberState` is a new type that holds more information about the subscriber (owner vs teammate, etc), and it is threaded into all of our middleware now instead of a bare bones subscription status.